send.go

```diff
@@ -29,6 +29,8 @@ import (
 	"strconv"
 	"strings"
 	"time"
+
+	"github.com/cenkalti/backoff"
```

Comment:
According to the documentation of the library this should be `github.com/cenkalti/backoff/v4`.